ckanext/s3filestore/uploader.py

```diff
@@ -145,7 +145,7 @@ def clear_key(self, filepath):
         except Exception as e:
             raise e
 
-    def get_signed_url_to_key(self, key_path):
+    def get_signed_url_to_key(self, key, extra_params={}):
```

Comment:
what does extra_params do?

Reply:
I put it there to align with the Keitaro fork. It's an optional way to override parameters when calling this function (it does a dict update).